maxim/logger/crewai/client.py

```
@@ -781,6 +852,26 @@ def maxim_wrapper(self, *args, **kwargs):
+                # Important to set and retrieve internal variables
+                if hasattr(self, "_eval_config") and \
+                    isinstance(self._eval_config, MaximEvalConfig):
+                    # Create a new dictionary for evaluation variables
+                    print(f"[GENERATION] self config: {vars(self._eval_config)}")
+                    eval_vars = {}
+                    # Add any existing variables from config
+                    for var_dict in self._eval_config.additional_variables:
+                        eval_vars.update(var_dict)
+                    # Add input and output
+                    if hasattr(generation, "_input"):
+                        eval_vars["input"] = getattr(generation, "_input")
```

Ruff (0.12.2)

Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)